crates/xilem_web/web_examples/mathml_svg/src/main.rs

```diff
@@ -83,8 +88,7 @@ pub fn main() {
                 ml::msup((ml::mi(format!("{}", t.b)), ml::mn("2"))),
                 ml::mo("="),
                 ml::msup((ml::mi(format!("{c}")), ml::mn("2"))),
-            )))
-            .attr("style", "width: 100%"),
+            ))),
```

Comment:
What happened tot his one?

Reply:
Well, `style` is not supported by `web_sys` currently for MathML elements (and thus by xilem_web, as it relies on web_sys currently), I don't think this had an effect anyways.

But I looked the [relevant interface](https://developer.mozilla.org/en-US/docs/Web/API/MathMLElement) up, which indeed supports `style`, so I guess when we don't want to specially handle this (and I think it will require some very special handling and likely additional wasm bloat for something that is very seldom used), it needs to be implemented in web_sys via an additional webidl interface (shouldn't be too hard though).